feat(optr-26228): add new options to prefilledou app setting

tomiolah opened the pull request:
- updated `App:Returns:PrefilledOrganizationUnit` app setting type, added new options, reference ticket and updated the description of the setting

omeaart commented:
@tomiolah can you add the option "Converted": true ? As this is changing type already, string only seems to be the way to go.
Converted -> the BE won't try to parse it anymore as the type

tomiolah commented:
> @tomiolah can you add the option "Converted": true ? As this is changing type already, string only seems to be the way to go. Converted -> the BE won't try to parse it anymore as the type

Added it 👍

coenvdwel reviewed (approved):
if you are certain that type-switch will not cause production issues, you can merge it 👌🏻
